compiler/compile/OMRCompilation.cpp

```diff
@@ -787,11 +787,16 @@ OMR::Compilation::getOSRInductionOffset(TR::Node *node)
    if (self()->getOSRTransitionTarget() != TR::postExecutionOSR)
       return 0;
 
+   if (node->getOpCodeValue() == TR::treetop || node->getOpCode().isCheck())
```

Comment:
"This change explicitly adds checks for OSR points that are calls"....um, where is the explicit check for a call ? I only see us checking the treetop node for opcodes that would be above calls.